### ISSUE TITLE
[Z-Wave] Polling an unreachable device blocks z-wave chip entirely for long periods

### ISSUE THREAD

**Issue by RobinBol** [athombv/homey#861]
**What is your Homey version (Settings → System)?**

0.9.2

**What did you try to achieve?**

Use the Greenwave powernode 6 and its polling feature.

**What was the result?**

The z-wave chip becomes unresponsive (greyed out z-wave settings), and never comes back.

**What did you expect as result?**

The z-wave chip to be available in between polling tries (polling happens every 120 seconds, so in between it should not be unavailable).

**Any other remarks**

After fixing the z-wave network by adding another powered (routing) node, the problem was resolved as most likely the powernode 6 became reachable through the added routing node.

TLDR; Polling unreachable devices blocks z-wave chip entirely for long periods.

This issue is created from https://github.com/athombv/com.greenwavesystems/issues/2, and placed here as it is a generic issue (not specific to the Greenwave app).

**Comment by WeeJeWel** [comment]
Duplicate of #851
